src/Features/Core/Portable/CodeFixes/FixAllOccurrences/IFixMultipleOccurrencesService.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Threading;
+using Microsoft.CodeAnalysis.Host;
+
+namespace Microsoft.CodeAnalysis.CodeFixes
+{
+    internal interface IFixMultipleOccurrencesService : IWorkspaceService
+    {
+        /// <summary>
+        /// Computes the fix muliple occurrences code fix for the given diagnostics with source locations, brings up the preview changes dialog for the fix and
+        /// apply the code action operations corresponding to the fix.
+        /// </summary>
+        void ComputeAndApplyFix(
+            ImmutableDictionary<Document, ImmutableArray<Diagnostic>> diagnosticsToFix,
+            Workspace workspace,
+            CodeFixProvider fixProvider,
+            FixAllProvider fixAllProvider,
+            string equivalenceKey,
+            string previewChangesTitle,
+            string waitDialogMessage,
+            CancellationToken cancellationToken);
+
+        /// <summary>
+        /// Computes the fix muliple occurrences code fix for the given diagnostics without any source location, brings up the preview changes dialog for the fix and
+        /// apply the code action operations corresponding to the fix.
+        /// </summary>
+        void ComputeAndApplyFix(
+            ImmutableDictionary<Project, ImmutableArray<Diagnostic>> diagnosticsToFix,
+            Workspace workspace,
+            CodeFixProvider fixProvider,
+            FixAllProvider fixAllProvider,
+            string equivalenceKey,
+            string previewChangesTitle,
+            string waitDialogMessage,
+            CancellationToken cancellationToken);
+    }
+}
```

Comment:
by the way, I think new concepts introduced here are

1. applying fix without view (all existing fix start from LB which require view)
2. applying multiple fixes at once (still only works over same fixer - in this case, suppression fixer - like FixAllProvider I guess?)

are those thing re-usable so that we can ask editor team to add LB support for error list?